pkg/ebpf/compiler/compiler.go

```diff
@@ -140,8 +141,13 @@ func NewEBPFCompiler(headerDirs []string, verbose bool) (*EBPFCompiler, error) {
 	} else {
 		dirs, err = kernel.FindHeaderDirs()
 		if err != nil {
-			ebpfCompiler.Close()
-			return nil, fmt.Errorf("unable to find kernel headers: %w", err)
+			log.Infof("unable to find kernel headers: %w. Attempting to download kernel headers", err)
```

Comment:
I believe `%w` only has meaning within `fmt.Errorf`, so this could be changed to `%s`.